### PR TITLE
Use Python 3.7 compatible dependencies and flask-session fix

### PR DESCRIPTION
**Pull request by apakulov-stripe**
Fixing two things for our environment:
- Python 3.7 compatible dependencies (gevent, greenlet and flask-session)
- Fixing flask_session import

r? @elewis-stripe

### CODE REVIEW

File: confidant/routes/v1.py [hunk 1]
```diff
@@ -669,6 +669,10 @@ def _pair_key_conflicts_for_services(_id, credential_keys, services):
     return conflicts
 
 
+def _lowercase_credential_pairs(credential_pairs):
+    return {i.lower(): j for i, j in credential_pairs.items()}
+
```
Comment (elewis-stripe) [inline confidant/routes/v1.py]: I think we actually removed this deliberately, unless it's breaking something? The original commit references a specific PR which still hasn't been merged yet: 
https://github.com/stripe/confidant/commit/bf261daca4f6dbdc736b78d654ff76eda8658ece
https://github.com/lyft/confidant/issues/103

**Review by elewis-stripe — CHANGES_REQUESTED** [review]
Just the one comment, the rest seems good to me

File: confidant/routes/v1.py (hunk 1 above)
Comment (apakulov-stripe) [reply confidant/routes/v1.py]: Yeah, my bad, removed in c73a70d

**Review by elewis-stripe — APPROVED** [review]
LGTM

**Pull request merged**